spec/loadbalancer/README.md

```diff
@@ -22,11 +25,74 @@ enterpriseconnect/loadbalancer:v1.1beta
 docker run -p 8080:80 --env-file $(pwd)/machine.env enterpriseconnect/loadbalancer:v1.1beta
 ```
 
+#### available tags
+- [```v1.1beta```](https://github.com/Enterprise-connect/oci/blob/v1.1beta/spec/loadbalancer/Dockerfile)
+- v1.1
+
+```sh
+docker pull enterpriseconnect/loadbalancer:v1.1beta
+```
+
+#### tag usage
+- [```v1.1```](https://github.com/Enterprise-connect/sdk/tree/v1.1/dist/agent)
+- [```v1```](https://github.com/Enterprise-connect/sdk/tree/v1/dist)
+- [```v1.1beta```](https://github.com/Enterprise-connect/sdk/tree/v1.1beta/dist/agent)
+- [```v1.1```](https://github.com/Enterprise-connect/sdk/tree/v1beta/dist) 
+
+
 ### Use-case I vm-2-vm
-vm to vm load-balancing for agent file trasfer.
+![LB High Level](/doc/lb-model.png)
```

Comment:
Please give it a short high-level introduction of what of this use-case about

Reply:
Added high level focus of use case